extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1333,7 +1333,7 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @return {boolean} whether html creatives should be sandboxed. */
   sandboxHTMLCreativeFrame() {
-    return isExperimentOn(this.win, 'sandbox-ads');
+    return true;
```

Comment:
I am a bit confused how the experiment flag works before because I'm seeing ad network override `#sandboxHTMLCreativeFrame()` anyway.